### PR TITLE
Optimizations, Refactor and Keval

### PR DESCRIPTION
**Pull request by notKamui**
Done some refactoring and small optimizations ;
Added (k)eval command.

IMPORTANT NOTICE in [this commit](https://github.com/SlamaFR/UGE-Bot/commit/876b89eb041637c0ad5a21e34d323666f19e59f5)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .idea/
 .gradle/
+gradle/wrapper/
 build/
```
Comment (SlamaFR) [inline .gitignore:6]: Should ignore log/ directory.

File: build.gradle [hunk 2]
```diff
@@ -29,6 +29,8 @@ dependencies {
     compile 'org.json:json:20200518'
     compile 'org.redisson:redisson:3.13.5'
     compile 'org.simplejavamail:simple-java-mail:4.1.1'
+
+    implementation 'com.notkamui.libs:keval:0.7.3'
```
Comment (SlamaFR) [inline build.gradle]: Should be a `compile` clause to ensure library is not missing when deploying

Comment (notKamui) [reply build.gradle]: Got it, done

**Review by SlamaFR — APPROVED** [review]
Seems good to me.

**Pull request merged**